sonic_y_cable/y_cable_base.py

```diff
@@ -616,15 +643,35 @@ def create_port(self, speed, fec_mode=FEC_MODE_NONE, anlt=False):
                 50000 -> 50G
                 100000 -> 100G
 
-            fec_mode:
-                One of the following predefined constants, the actual fec mode for the port to be configured:
+            fec_mode_tor_a:
+                One of the following predefined constants, the actual fec mode for the tor A to be configured:
```

Comment:
s/fec/FEC/

Please make this change throughout all docstrings for consistency

Reply:
fixed